client/rest-high-level/src/main/java/org/elasticsearch/client/ml/job/config/Job.java

```diff
@@ -55,7 +55,7 @@
     public static final ParseField DAILY_MODEL_SNAPSHOT_RETENTION_AFTER_DAYS =
         new ParseField("daily_model_snapshot_retention_after_days");
     public static final ParseField RESULTS_RETENTION_DAYS = new ParseField("results_retention_days");
-    public static final ParseField ANNOTATIONS_RETENTION_DAYS = new ParseField("annotations_retention_days");
+    public static final ParseField SYSTEM_ANNOTATIONS_RETENTION_DAYS = new ParseField("system_system_annotations_retention_days");
```

Comment:
```suggestion
    public static final ParseField SYSTEM_ANNOTATIONS_RETENTION_DAYS = new ParseField("system_annotations_retention_days");
```

Reply:
Thanks for noticing. This and other "typos" have already been covered in "Fix typo" commit.